src/test/kotlin/no/nav/syfo/behandlerdialog/AvvistMeldingSpek.kt

```diff
@@ -68,7 +71,56 @@ class AvvistMeldingSpek : Spek({
 
                 val personOppgave = database.connection.getPersonOppgaveByReferanseUuid(
                     referanseUuid = referanseUuid,
-                )!!.toPersonOppgave()
+                ).map { it.toPersonOppgave() }.first()
+                personOppgave.publish shouldBeEqualTo false
+                personOppgave.type.name shouldBeEqualTo PersonOppgaveType.BEHANDLERDIALOG_MELDING_AVVIST.name
+
+                val producerRecordSlot = slot<ProducerRecord<String, KPersonoppgavehendelse>>()
+                verify(exactly = 1) {
+                    producer.send(capture(producerRecordSlot))
+                }
+
+                val kPersonoppgavehendelse = producerRecordSlot.captured.value()
+                kPersonoppgavehendelse.hendelsetype shouldBeEqualTo PersonoppgavehendelseType.BEHANDLERDIALOG_MELDING_AVVIST_MOTTATT.name
+                kPersonoppgavehendelse.personident shouldBeEqualTo UserConstants.ARBEIDSTAKER_FNR.value
+            }
+            it("stores avvist melding from kafka as oppgave in database and publish as new oppgave also when ubesvartopphgave exists for same referanseUuid") {
```

Comment:
Dette er en ny test for å sjekke at koden tåler at det finnes en oppgave på referanse_uuid'en fra før.